solr/modules/ltr/src/java/org/apache/solr/ltr/response/transform/LTRFeatureLoggerTransformerFactory.java

```diff
@@ -241,11 +243,23 @@ public void setContext(ResultContext context) {
      *
      * @param transformerFeatureStore the explicit transformer feature store
      */
-    private LoggingModel createLoggingModel(String transformerFeatureStore) {
+    private LoggingModel createLoggingModel(
+        String transformerFeatureStore, Boolean docsWereNotReranked) {
       final ManagedFeatureStore fr = ManagedFeatureStore.getManagedFeatureStore(req.getCore());
 
+      if (transformerFeatureStore != null) {
+        fr.checkMissingFeatureStore(transformerFeatureStore);
```

Comment:
maybe I would return a boolean with true if missing, false otherwise and then through the exception if missing

Reply:
I will change the method name and try your suggestion, modifying the testWrongFeatureStore() in TestManagedFeatureStore.java accordingly.